queries/prql/highlights.scm

```diff
@@ -0,0 +1,140 @@
+[
+  (keyword_from)
+  (keyword_filter)
+  (keyword_derive)
+  (keyword_group)
+  (keyword_aggregate)
+  (keyword_sort)
+  (keyword_take)
+  (keyword_window)
+  (keyword_join)
+  (keyword_select)
+  (keyword_switch)
+  (keyword_inner)
+  (keyword_right)
+  (keyword_full)
+  (keyword_in)
+  (keyword_rolling)
+  (keyword_rows)
+  (keyword_expanding)
+  (keyword_let)
+  (keyword_prql)
+  (keyword_switch)
+  (keyword_append)
+  (keyword_remove)
+  (keyword_intersect)
+] @keyword
+
+(literal) @string
+
+(assignment
+  alias: (field) @field)
+
+alias: (identifier) @field
+
+(f_string) @string.special
+(s_string) @string.special
+
+(comment) @comment
+
+(keyword_func) @keyword.function
+
+(function_call
+  (identifier) @function.call)
+
+[
+  "+"
+  "-"
+  "*"
+  "/"
+  "="
+  "=="
+  "<"
+  "<="
+  "!="
+  ">="
+  ">"
+  "->"
+  (bang)
+] @operator
+
+[
+  "("
+  ")"
+  "["
+  "]"
+] @punctuation.bracket
+
+[
+  ","
+  "."
+  (pipe)
+] @punctuation.delimiter
+
+(literal
+  (integer) @number)
+
+(integer) @number
+
+(literal
+  (decimal_number) @float)
+
+(decimal_number) @float
+
+[
+  (keyword_min)
+  (keyword_max)
+  (keyword_count)
+  (keyword_count_distinct)
+  (keyword_average)
+  (keyword_avg)
+  (keyword_sum)
+  (keyword_stddev)
+  (keyword_count)
+] @function.call
```

Comment:
I don't know. There are so few functions/builtin functions that I do not want to distinguish between these by color.

Reply:
I think that writing your queries based on your colorschemes highlighting is not the right way to go about this. If that's a concern of yours, you can link the highlight groups for that specific filetype in your config.

Please be as correct as possible with the queries :)

Reply:
No, that was not the concern. The "problem" with languages like PRQL or SQL is that you have a lot of keyworks that are kind of function call and they follow each other. Thus everything is in the same color and therefore useless. The contribution guideline also states that since languanges are vastly different one could be a bit more free in choosing the "marker" like `@function.call`

Reply:
I believe `@function.call` is fine here.

Reply:
>  The "problem" with languages like PRQL or SQL is that you have a lot of keyworks that are kind of function call and they follow each other.

Thanks for the clarification, this is ok then